README.md

```diff
@@ -78,6 +78,12 @@ export FAUNA_ROOT_KEY='test%40faunadb.com:secret'
 rake test
 ```
 
+To run a single test, use e.g. `ruby -Ilib:test test/client_test.rb`.
+
+## Documenting
+
+Use `bundle exec rake rdoc` to generate documentation.
```

Comment:
I think we can skip `bundle exec` here. End users of this library should probably already know when to add that to ruby commands.